### PR TITLE
Miocroad Bid Adapter: Added Support for Audience IDs

### PR DESCRIPTION
**Pull request by azuryo**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Support Audience ID below in MicroAd Bid Adapter
- [IM-UID](https://docs.prebid.org/dev-docs/modules/userId.html#im-uid-by-intimate-merger)
- [ID5 ID](https://docs.prebid.org/dev-docs/modules/userId.html#id5-universal-id)
- [Unified ID](https://docs.prebid.org/dev-docs/modules/userId.html#unified-id)
- [Novatiq Hyper ID](https://docs.prebid.org/dev-docs/modules/userId.html#novatiq-hyper-id)
- [Parrable ID](https://docs.prebid.org/dev-docs/modules/userId.html#parrable-id)
- [AudienceOne ID](https://docs.prebid.org/dev-docs/modules/userId.html#audienceone-id-by-dac)
- [Ramp ID](https://docs.prebid.org/dev-docs/modules/userId.html#rampid)
- [Criteo ID](https://docs.prebid.org/dev-docs/modules/userId.html#criteo-id-for-exchanges)
- [Shared ID](https://docs.prebid.org/dev-docs/modules/userId.html#sharedid)

contact email of the adapter’s maintainer
- [x] official adapter submission
- contact email of the adapter’s maintainer: [prebid@microad.co.jp](prebid@microad.co.jp)

### CODE REVIEW

**Comment by azuryo** [comment]
@Iksharma

Thanks for the feedback. 
Ran `gulp test-coverage` and results for our adapter state 100% for statements, branches, functions and lines.
For reference, I'm attaching the results from `gulp test-coverage` and `gulp view-coverage`.
Am I looking at the results incorrectly? スクリーンショット 2022-07-19 14 19 06

**Comment by lksharma** [comment]
@azuryo that was my mistake your test coverage is 100%.

File: test/spec/modules/microadBidAdapter_spec.js [hunk 1]
```diff
@@ -266,35 +266,70 @@ describe('microadBidAdapter', () => {
       });
     });
 
-    it('should add Liveramp identity link if it is available in request parameters', () => {
+    it('should not add Liveramp identity link and Audience ID if it is not available in request parameters', () => {
       const bidRequestWithLiveramp = Object.assign({}, bidRequestTemplate, {
```
Comment (lksharma) [inline test/spec/modules/microadBidAdapter_spec.js:270]: should `bidRequestWithLiveramp` be changed to `bidRequestWithLiveramp`?

Comment (azuryo) [reply test/spec/modules/microadBidAdapter_spec.js]: @lksharma 
Fixed - https://github.com/prebid/Prebid.js/pull/8662/commits/9ed2ded075d730a305e8cb6af186c56c28258f9c

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**